rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -354,6 +357,53 @@ void buildMigrationTransaction_withAFederationWithEnoughUTXOs_beforeRSKIP376_sho
         assertEquals(BTC_TX_VERSION_1, migrationTransactionUnsigned.getVersion());
     }
 
+    @Test
+    void createPegoutTransaction_with210Inputs_with50Outputs_shouldHaveASizeAboveMaximumBtcStandardSize() {
+        // arrange
+        int numberOfInputs = 210;
+
+        // act
+        BtcTransaction pegoutTx = createPegoutTransactionWith50Outputs(p2shP2wshErpProposedFederation, keys, numberOfInputs);
+        int size = calculateSignedSegwitBtcTxVirtualSize(pegoutTx);
+
+        // assert
+        assertTrue(size > BtcTransaction.MAX_STANDARD_TX_SIZE);
+    }
+
+    @Test
+    void createPegoutTransaction_with150Inputs_with50Outputs_shouldHaveASizeBelowMaximumBtcStandardSize() {
```

Comment:
So, if you create a tx with 150 inputs and 550 outputs, would it exceed the MAX_STANDARD_SIZE? I guess both inputs and outputs increase the vsize, right? It would be helpful to include a comment or additional test cases that help readers understand the reasoning behind the values [150,200] and [500]. Wdyt?